src/thor/dijkstras.cc

```diff
@@ -694,29 +699,22 @@ void Dijkstras::ComputeMultiModal(
   mode_ = mode;
   const auto& tc = mode_costing[static_cast<uint8_t>(travel_mode_t::kPublicTransit)];
 
-  // Get maximum transfer distance
-  // TODO - want to allow unlimited walking once you get off the transit stop...
-  max_transfer_distance_ = 99999.0f; // costing->GetMaxTransferDistanceMM();
-
-  // For now the date_time must be set on the origin.
-  if (origin_locations.Get(0).date_time().empty()) {
-    LOG_ERROR("No date time set on the origin location");
-    return;
-  }
+  // Get maximum transfer and walking distance
+  max_transfer_distance_ = pc->GetMaxTransferDistanceMM();
+  max_walking_dist_ =
+      options.costings().find(Costing::pedestrian)->second.options().transit_start_end_max_distance();
```

Comment:
not so sure about these two.. these checks were basically disabled before. I still do think that also with isochrones we might want to respect the relevant costing options. WDYT @gknisely ?